internal/pkg/template/repository/manager/manager.go

```diff
@@ -0,0 +1,65 @@
+package manager
+
+import (
+	"crypto/sha256"
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
+	"github.com/keboola/keboola-as-code/internal/pkg/git"
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/template/repository"
+)
+
+type Manager struct {
+	logger       log.Logger
+	repositories map[string]*GitRepository
+}
+
+type GitRepository struct {
+	Dir string
+	Fs  filesystem.Fs
+}
+
+func hashRepositoryName(repo model.TemplateRepository) string {
+	h := sha256.New()
+	h.Write([]byte(fmt.Sprintf("%s:%s", repo.Url, repo.Ref)))
+	return string(h.Sum(nil))
+}
+
+func New(logger log.Logger) (*Manager, error) {
+	m := &Manager{
+		logger:       logger,
+		repositories: make(map[string]*GitRepository),
+	}
+	err := m.AddRepository(repository.DefaultRepository())
+	if err != nil {
```

Comment:
Toto sa da spojit na jeden riadok, musim pozriet, preco to linter reportuje iba niekedy.